src/main/java/mate/academy/spring/dao/RoleDao.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.spring.dao;
+
+import java.util.Optional;
+import mate.academy.spring.model.Role;
+
+public interface RoleDao {
+    Role add(Role role);
+
+    Optional<Role> getByName(Role.RoleName name);
```

Comment:
We should pass String roleName
```suggestion
    Optional<Role> getByName(String roleName);
```